src/FakeItEasy/Configuration/IThenConfiguration.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace FakeItEasy.Configuration
+{
+    using System.Diagnostics.CodeAnalysis;
+
+    /// <summary>
+    /// Configures the next behavior for this call.
+    /// </summary>
+    /// <typeparam name="TInterface">The type of configuration interface to return.</typeparam>
+    public interface IThenConfiguration<out TInterface>

```

Comment:
Some other configuration interfaces that lack `IHideObjectMembers`:

- `IDoNothingConfiguration<TInterface>`
- `IOutAndRefParametersConfiguration<TInterface>`
- `IWhereConfiguration<out T>` (which probably needs to be updated anyway, because the doc comment about `T` is incorrect)

Should I add it to them as well, or do it in a separate PR?